keras_core/backend/jax/distribution_lib.py

```diff
@@ -0,0 +1,56 @@
+"""!!!DO NOT USE!!!
+
+Distribution related class for JAX backend.
+
+This is just a prototype and we might want to unify it
+with other backends in the future.
+"""
+import jax
+
+
+def list_devices(device_type=None):
+    """Return all the available devices based on the device type.
+
+    Note that this should return the global devices in a distributed setting.
+
+    Args:
+        device_type: string of `"CPU"`, `"GPU"` or `"TPU"`. Default to `GPU` or
```

Comment:
We normalize to lowercase and we generally use lowercase for string constants everywhere, so let's them lowercase in the docstring

Reply:
i see. tf.dtensor was using upper case for device type, but jax was using lower case.